libmcount/plthook.c

```diff
@@ -455,7 +455,13 @@ static int setup_mod_plthook_data(struct dl_phdr_info *info, size_t sz, void *ar
 		"linux-gate.so.1",
 		"ld-linux-*.so.*",
 		"libdl.so.2",
-		"libdl-2.*.so"
+		"libdl-2.*.so",
+		/* additional misc libraries */
+		"libstdc++.so.*",
+		"libelf.so.*",
+		"libdw.so.*",
+		"libSegFault.so",
+		"libunwind*"
```

Comment:
That means you cannot see PLT calls from these libraries.  I guess it depends on the use-case unless the library is a very primitive one.  At least we want to see C++ implementation details, right?

Reply:
I saw some broken addresses which don't matched with symbols while tracing chrome and `libstdc++.so` is always loaded because of our full demangler even for C programs. I think we better make it safe first, then we can opt them out if we really need to see their functions in a safe manner.

Reply:
One more thing is that most of STL implementation is in its header files, not in the library so I don't find any problem analyzing C++ standard libraries even without `libstdc++.so`.

Reply:
@namhyung You're right. `libstdc++.so` shouldn't be skipped. I will remove it from the skip list.